scrapy/spiderloader.py

```diff
@@ -94,6 +105,8 @@ def find_by_request(self, request: Request) -> list[str]:
         """
         Return the list of spider names that can handle the given request.
         """
+        if self.load_from_path:
+            raise NotImplementedError("Not supported")
```

Comment:
This function, via `scrapy.utils.spider.spidercls_for_request()`, is used by `scrapy fetch`, `scrapy parse` and `scrapy shell` to find which spider to use. Can you please check what happens when the new setting is enabled and how to make the user experience better, if needed? Thanks.